packages/mdc-textfield/mdc-text-field.scss

```diff
@@ -173,6 +171,17 @@
     stroke-width: 2px;
   }
 
+  &:not(.mdc-text-field--invalid) {
```

Comment:
Is this fixing a different issue?

Reply:
Yeah, I moved all the spec changes over to #1855